src/index.html

```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="#home">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
 
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+            <a href="#apple" class="nav__link">Apple</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Samsung</a>
+            <a href="#samsung" class="nav__link">Samsung</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Смарфоны и телефоны</a>
+            <a href="#смартфоны#и#телефоны" class="nav__link">Смарфоны и телефоны</a>
```

Comment:
The following code style is more readable
```suggestion
            <a 
              href="#смартфоны#и#телефоны" 
              class="nav__link"
            >
              Смарфоны и телефоны
            </a>
```